docs/migrating-v1beta1-to-v1.md

```diff
@@ -7,15 +7,21 @@ weight: 4000
 
 # Migrating From Tekton `v1beta1` to Tekton `v1`
 
+- [Upgrading objects to new storedversion](#upgrading-objects-to-new-storedversion)
 - [Changes to fields](#changes-to-fields)
 - [Upgrading `PipelineRun.Timeout` to `PipelineRun.Timeouts`](#upgrading-pipelinerun.timeout-to-pipelinerun.timeouts)
 - [Deprecating Resources from Task, TaskRun, Pipeline and PipelineRun](#deprecating-resources-from-task,-taskrun,-pipeline-and-pipelinerun)
 - [Replacing `taskRef.bundle` and `pipelineRef.bundle` with Bundle Resolver](#replacing-taskRef.bundle-and-pipelineRef.bundle-with-bundle-resolver)
 - [Adding `TaskRunTemplate` in `PipelineRun.Spec`](#adding-taskruntemplate-to-pipelinerun.spec)
 
 
-This document describes the differences between `v1beta1` Tekton entities and their
-`v1` counterparts. It also describes the changed fields and the deprecated fields into v1.
+This document serves as the guide for the migration from `v1beta1` to `v1` apiVersion.
+It identifies the differences between `v1beta1` Tekton objects and their `v1` counterparts.
+It also describes the changed fields and the deprecated fields moving to v1.
+
+## Upgrading objects to new storedversion
```

Comment:
I think we wouldn't want a change in stored version to be transparent to users. It sounded like the docs Vincent mentioned wanting were docs on how to identify whether there are any v1beta1 objects remaining on the cluster, so they can be removed or swapped to v1. Once we have a storage version migrator in place we can also add instructions on how to use that (if any are needed).

Reply:
> I think we wouldn't want a change in stored version to be transparent to users.

Will we want to recommend users to specify v1 CRDs instead of v1beta1 since then? curious because right now the CRD releases only provided with v1 but not yet ask so.

>  It sounded like the docs Vincent mentioned wanting were docs on how to identify whether there are any v1beta1 objects remaining on the cluster, so they can be removed or swapped to v1

Thanks for clarifications. Sounds like this should be in scope of this PR while we could leave the migrator parts till later when we decide on when we deprecate v1beta1? 🤔

Reply:
Yes, I think we should recommend users specify v1 CRDs instead of v1beta1. I think we can accomplish this by having our docs use v1 examples (https://github.com/tektoncd/pipeline/pull/6414).

Adding docs for identifying v1beta1 CRDs on the cluster in this PR sounds like a great idea, maybe you and @vdemeester could work together on that?

And yes, we can leave the storage version migrator for later.